mindsdb/streams/redis/redis_stream.py

```diff
@@ -25,12 +26,83 @@ def __init__(self, host, port, database, stream_in, stream_out, predictor, _type
 
         self.stop_event = Event()
         self.company_id = os.environ.get('MINDSDB_COMPANY_ID', None)
-        super().__init__(target=RedisStream.make_prediction, args=(self,))
-
+        if self._type == 'timeseries':
+            super().__init__(target=RedisStream.make_predictions, args=(self,))
```

Comment:
Isn't this backwards? i.e. `if self._type == 'timeseries'` then `target=RedisStream.make_timeseries_predictions`?

Reply:
Thanks, @paxcema  It is the typo. Fixed.